src/h_vialib/client.py

```diff
@@ -46,16 +50,22 @@ def __init__(self, service_url, host_url):
             "via.external_link_mode": "new-tab",
         }
 
-    def url_for(self, doc):
+    def url_for(self, url, content_type=None):
         """Generate a Via url to proxy `doc`.
 
-        :param doc a ViaDoc representation of a resource
+        :param url: URL to proxy thru Via
+        :param content_type: content type, if known, of the document
+        :return: Full via url to proxy `url` including signature
+        :rtype: str
```

Comment:
I think on the whole we leave out type declarations, because they just clutter up the place